host/xtest/install_ta.c

```diff
@@ -45,15 +45,20 @@ static void *read_ta(const char *dname, const char *fname, size_t *size)
 		err(1, "fseek");
 
 	s = ftell(f);
+	if (s < 0) {
+		fclose(f);
```

Comment:
maybe `err(1, "ftell");` is more appropriate as we don't expect `ftell` to fail.

Reply:
At first glance seems to be appropriate.
I will leave to Rajesh to comment.